nixos/modules/services/cluster/dynomite.nix

```diff
@@ -0,0 +1,82 @@
+{ config, pkgs, lib, ... }:
+
+with lib;
+let
+  cfg = config.services.dynomite;
+  configFile = pkgs.writeText "dynomite.yaml" "${ cfg.config }";
+in {
+  options = {
+    services.dynomite = {
+      enable = mkEnableOption "dynomite server";
+       config = mkOption {
+        default = "";
+        example = ''
+          dyn_o_mite:
+            listen: 127.0.0.1:8102
+            dyn_listen: 127.0.0.1:8101
+            tokens: '101134286'
+            servers:
+            - 127.0.0.1:6379:1
+            data_store: 0
+            mbuf_size: 16384
+            max_msgs: 300000
+        '';
+        type = types.lines;
+        description = ''
+          Configuration for dynomite, see <link xlink:href="https://github.com/Netflix/dynomite"/>
+        '';
+      };
+      # Enable as soon as dynomite supports YAML 1.2
```

Comment:
- Is there an upstream issue tracking this?
- Is there any simple tools that could convert our generated yaml into a format `dynomite` can understand?

Reply:
i worked around it